stackdriver/client.go

```diff
@@ -180,6 +180,21 @@ func (c *Client) Store(req *monitoring.CreateTimeSeriesRequest) error {
 					return
 				}
 				switch status.Code() {
+				// codes.DeadlineExceeded:
+				//   It is safe to recover and retry
+				//   for all cases of
+				//   google.monitoring.v3.MetricService.CreateTimeSeries.
```

Comment:
Add the reasoning as to why it is safe to retry. For example,

> It is safe to recover and retry `google.monitoring.v3.MetricService.CreateTimeSeries` requests because the client enforces in-order writes, which is a requirement for Stackdriver monitoring writes.

Reply:
edit: "retry with backoff" like in the `codes.Unavailable` section

Reply:
Updated. Please take another look.